src/entities/note.cpp

```diff
@@ -2277,17 +2277,36 @@ QString Note::textToMarkdownHtml(QString str, const QString &notesPath,
        src=\")((?!file:\/\/).+)\")"),
                 "\\1file://" + windowsSlash + notePath + "/\\2\"");
     */
-
-    const QString fontString =
-        settings.value(QStringLiteral("MainWindow/noteTextView.code.font"))
+    const QString previewFontString =
+        settings.value(QStringLiteral("MainWindow/noteTextEdit.font"))
             .toString();
+    QString margin_listitem_str = "";
+    QString margin_checkbox_str = "";
+    if (!previewFontString.isEmpty()) {
+        QFont previewFont;
+        previewFont.fromString(previewFontString);
+        const int width_checkbox = QFontMetrics(previewFont).horizontalAdvance("☑"); // &#9744; and &#9745;
+        const int width_bullet = QFontMetrics(previewFont).horizontalAdvance("•"); // &#8226;
```

Comment:
Are you sure that this bullet is the bullet used in lists?

Reply:
> I guess you have that setting on:
> 


Yes, that was on! But it does change the font...
![image](https://user-images.githubusercontent.com/6073574/83939728-7757a680-a7df-11ea-8b3e-a362f9dff7e8.png)

Reply:
> Yes, that was on! But it does change the font...

see https://github.com/pbek/QOwnNotes/pull/1771/files#r436251486

Reply:
What do you mean specifically?

Reply:
> Are you sure that this bullet is the bullet used in lists?

I hope so:
https://www.fileformat.info/info/unicode/char/2022/index.htm

Reply:
> What do you mean specifically?

Try https://github.com/pbek/QOwnNotes/pull/1771/files#r436251486 again, you have wrongly "resolved the conversation" (because you smuggled your RTL comments into that conversation), and because of that the html anchor didn't work any more because GitHub hid the conversation.

Reply:
For example your `Yes, that was on! But it does change the font...` also doesn't belong to this conversation. So we need to be careful of what we "resolve". :grin: